docs/release-notes/release-notes-0.16.0.md

```diff
@@ -11,6 +11,12 @@ transaction](https://github.com/lightningnetwork/lnd/pull/6730).
   allow specifying a root key for macaroons during wallet init rather than
   having lnd randomly generate one for you.
 
+* [A new `SignedInputs` field is added to `SignPsbtResponse` that returns the
+  indices of inputs that were signed by our wallet. Prior to this change
+  `SignPsbt` didn't indicate whether the Psbt held any inputs for our wallet
+  to sign.]
```

Comment:
AFAIK you can't have a newline between the two bracket types, otherwise GitHub won't render the link correctly.

Reply:
thanks again for flagging.